### PR TITLE
Minor changes to get fixed wing control apps running for posix

### PR DESCRIPTION
**Pull request by devbharat**

### CODE REVIEW

File: src/modules/fw_pos_control_l1/fw_pos_control_l1_main.cpp [hunk 1]
```diff
@@ -1994,7 +1997,7 @@ int fw_pos_control_l1_main(int argc, char *argv[])
 
 	if (!strcmp(argv[1], "stop")) {
 		if (l1_control::g_control == nullptr)
-			errx(1, "not running");
+			warnx("not running");
```
Comment (LorenzMeier) [inline src/modules/fw_pos_control_l1/fw_pos_control_l1_main.cpp]: return 1; missing here

File: src/modules/fw_pos_control_l1/fw_pos_control_l1_main.cpp [hunk 2]
```diff
@@ -2003,10 +2006,10 @@ int fw_pos_control_l1_main(int argc, char *argv[])
 
 	if (!strcmp(argv[1], "status")) {
 		if (l1_control::g_control) {
-			errx(0, "running");
+			warnx("running");
```
Comment (LorenzMeier) [inline src/modules/fw_pos_control_l1/fw_pos_control_l1_main.cpp]: return 0; missing

**Comment by LorenzMeier** [comment]
Nice! Let me know if I can assist further there - I would like to get fixed wing going as well.

**Comment by devbharat** [comment]
Yes...I was just about to say these were missing. Hold on!

File: src/modules/fw_pos_control_l1/fw_pos_control_l1_main.cpp [hunk 3]
```diff
@@ -1821,7 +1824,6 @@ FixedwingPositionControl::task_main()
 	warnx("exiting.\n");
 
 	_control_task = -1;
-	_exit(0);
```
Comment (LorenzMeier) [inline src/modules/fw_pos_control_l1/fw_pos_control_l1_main.cpp]: return 0 missing

File: src/modules/fw_att_control/fw_att_control_main.cpp [hunk 4]
```diff
@@ -1132,7 +1135,6 @@ FixedwingAttitudeControl::task_main()
 
 	_control_task = -1;
 	_task_running = false;
-	_exit(0);
```
Comment (LorenzMeier) [inline src/modules/fw_att_control/fw_att_control_main.cpp]: return 0 missing

**Comment by devbharat** [comment]
I think those last two returns shouldnt be there, they aren't in mc_pos_control. Its a void function, hence the error.

**Comment by LorenzMeier** [comment]
Applied to master, thanks!

**Pull request closed without merging**